src/main/features/directions-questionnaire/views/hearing-location.njk

```diff
@@ -0,0 +1,59 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, submitButton %}
+
+{% set heading = 'Choose a hearing location' %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <p>Tell us your preferred location for a hearing, in case the claim goes to one.</p>
+      <p>We’ll use the location you choose unless the claimant has an exceptional reason to object, for example accessibility needs.</p>
+      <p>You can choose any location - Guildford County Court is nearest to the home address you gave us.</p>
+
+      <h1 class="heading-medium">Do you want to choose Guildford County Court?</h1>
+
+      <div class="form-group">
+        <fieldset>
+         <div class="multiple-choice">
+           <input id="radio-court-yes" type="radio" name="court" value="yes">
+           <label for="radio-court-yes">Yes</label>
+         </div>
+         <div class="multiple-choice" data-target="courtsearch">
+           <input id="radio-court-no" type="radio" name="court" value="no">
+           <label for="radio-court-no">No</label>
+         </div>
+        </fieldset>
+      </div>
+
+      <div class="form-group js-hidden" id="courtsearch">
+        <h1 class="heading-medium">Choose another court</h1>
+        <fieldset>
+         <div class="multiple-choice" data-target="name">
+           <input id="radio-court-name" type="radio" name="court-search" value="name">
+           <label for="radio-court-name">Enter court name</label>
+         </div>
+         <div class="panel panel-border-narrow js-hidden" id="name">
+           <div class="form-group">
+             <label class="form-label" for="county_court">Court name</label>
+             <input type="text" id="county_court" name="county_court" class="form-control" style="width: 500px" />
```

Comment:
don't add style to inputs. please use classes. there should already be existing class to use

Reply:
njk removed